content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -148,7 +148,7 @@ <h3>Scaling a Deployment</h3>
         <div class="row">
             <div class="col-md-12">
                <h3>Load Balancing</h3>
-               <p>Let's check that the Service is load-balancing the traffic. To find out the exposed IP and Port we can use the describe service as we learned in the previous part of the tutorial:</p>
+               <p>Let's check that the Service is load-balancing the traffic. To find out the exposed IP and Port we can use the <tt>describe service</tt> subcommand as we learned in <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous part</a> of the tutorial:</p>
```

Comment:
I usually put the definite article outside hyperlinks:
```suggestion
               <p>Let's check that the Service is load-balancing the traffic. To find out the exposed IP and Port we can use the <tt>describe service</tt> subcommand as we learned in the <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"> previous part</a> of the tutorial:</p>
```